src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -154,18 +154,9 @@ protected virtual void ExtractQualifiedHeadings(List<string> keys, List<string>
          }
       }
 
-      protected string GetLastBracketsOfString(string header)
+     protected string GetAndValidateUnitFromBrackets(string unit, IReadOnlyList<IDimension> supportedDimensions)
       {
-         return Regex.Match(header, @"\[([^\]\[]*)\]$").Value;
-      }
-
-      protected string GetAndValidateUnitFromBrackets(string units, IReadOnlyList<IDimension> supportedDimensions)
-      {
-         return units
-            .Trim().Substring(1, units.Length - 2).Trim() //remove the brackets and whitespaces from end and beginning
-            .Split(',')                                   //we split in case there are more than one units separated with ,
```

Comment:
@Yuri05 @PavelBal @georgeDaskalakis @abdelr 
What is this here? Split unit with a "," ? Where did we specify something like this?

Reply:
I cannot remember us specifying something like this.

Reply:
We had already a discussion in the past about the unit extraction, and I thought the `.Split(',')` was removed...
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/pull/1204#pullrequestreview-712614163

Reply:
well actually Yuri proposes a solution without regex which reads much much better. I am going to implement it. 
When you solve a problem with regex, you have two problems :)